package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "next-compression-plugin",
+  "version": "1.0.0",
```

Comment:
We should start this as `0.0.0` and when we publish the package, we can publish as a new major version and it will automatically bump the version to `1.0.0`.